app/pathogen/arbovirus/dashboard/(visualizations)/median-seroprevalence-by-who-region-and-age-group-table.tsx

```diff
@@ -27,6 +28,49 @@ enum AgeGroup {
   "Multiple groups" = "Multiple groups",
 }
 
+interface GetMedianSeroprevalenceInformationFromDataInput {
+  data: any[];
+}
+
+interface GetMedianSeroprevalenceInformationFromDataOutput {
+  seroprevalenceNumber: number | undefined;
+  seroprevalencePercentageString: string;
+  backgroundColourHexCode: string;
+}
+
+const getMedianSeroprevalenceInformationFromData = (
```

Comment:
Moved from below into function for both the csv downloader and table to use. Something more elegant might involve them formatting their stuff together? :thinking: